ts/src/bitmex.ts

```diff
@@ -624,10 +629,12 @@ export default class bitmex extends Exchange {
                     'swap': swap,
                     'future': future,
                     'option': false,
+                    'index': index,
```

Comment:
I know we don't have them unified yet, but one-by-one we can add them in individual exchanges and then add it in docs eventually.